packages/nextjs/src/utils/initializeFronteggApp/index.ts

```diff
@@ -81,21 +80,20 @@ const initializeFronteggApp = ({
     hostedLoginBox: options.hostedLoginBox ?? false,
     disableSilentRefresh: options.authOptions?.disableSilentRefresh ?? true,
     user: userData,
-    tenantsState,
+    tenantsState: tenantsState as AuthState['tenantsState'],
   };
 
-  const sharedStore = createFronteggStore(
-    { context: contextOptions, appName: appName ?? 'default' },
+  const sharedStore = createStore({
```

Comment:
can you explain this change?

Reply:
no we are using the new valtio store, in order to support more readable options I migrated it to parameter object pattern instead of standard function parameters